src/components/footer/TheFooter.vue

```diff
@@ -133,4 +139,5 @@ de:
   imprint: "Impressum"
   privacyPolicy: "Datenschutz"
   termsOfUse: "AGB"
+  callUs: "Rufen Sie uns auf +44 123 456 000 an"
```

Comment:
As it breaks the footer a bit, I would just drop the whole number thingy. It's a fake number anyway, so let's remove all these fake things that make design more difficult (as we did with size guide).

Reply:
So, I'd leave the "contact us" from before and just remove the phone from the header.